README.md

```diff
@@ -4,10 +4,10 @@
  
 # pretty-print - pp
 
-Adds a `pp(Any?)` and `T.pp()` method to pretty print any Java or Kotlin object.
+Adds a `pp(Any?)` and `T.pp()` method to pretty print Java or Kotlin object.
 
-`pp(Any?)` takes any object and will print it in a pretty format.
-`T.pp()` can be called on any object inline and will allow you to pretty print inside of a method chain.
+`pp(Any?)` takes an object and prints in a pretty format.
```

Comment:
same here. I'd rather have `any`. If you remove `any` you'll also need to retain `it` as `prints in a pretty format` doesn't really indicate what is being printed.